code-studio/src/js/components/pairing.jsx

```diff
@@ -0,0 +1,256 @@
+/**
+ * A component for managing pair programming
+ */
+
+/* global React, dashboard */
+
+var SectionSelector = React.createClass({
+  handleChange: function (event) {
+    this.props.handleChange(event);
+  },
+
+  render: function () {
+    if (this.props.sections.length === 1) {
+      return null;
+    }
+
+    return (
+        <select name="sectionId" value={this.props.selectedSectionId} onChange={this.handleChange}>
+        <option key="blank" value="">Choose your section</option>
+        {
+          this.props.sections.map(function (section) {
+            return <option key={section.id} value={section.id}>{section.name}</option>;
+          })
+        }
+      </select>
+    );
+  },
+});
+
+var StudentSelector = React.createClass({
+  getInitialState: function () {
+    return {
+      selectedStudentIds: []
+    };
+  },
+
+  handleStudentClicked: function (event) {
```

Comment:
More of a high level question, but should we have something that prevents a student from selecting too many peers?